doc/source/tutorial-quickstart-tensorflow.rst

```diff
@@ -28,23 +35,31 @@ Since we want to use the Keras API of TensorFlow (TF), we have to install TF as
 Flower Client
 -------------
 
-Next, in a file called :code:`client.py`, import Flower and TensorFlow:
+Next, in a file called :code:`client.py`, import Flower, Flower Datasets, and TensorFlow:
 
 .. code-block:: python
 
     import flwr as fl
```

Comment:
We started to drop the `import flwr as fl` convention and started to import the required components directly